### PR TITLE
DOC-1847 and editorial revisions

### PR DESCRIPTION
**Pull request by aimurphy**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-integration/ci-technical-reference/harness-ci.md [hunk 1]
```diff
@@ -10,35 +10,30 @@ helpdocs_is_private: false
...
+
+* `harness/ci-addon`: Used to execute steps on containers in Kubernetes pods
+* `harness/ci-lite-engine`: Used to orchestrate execution of steps on a Kubernetes pods
+* `harness/drone-git`: Used to clone git repos
+* `plugins/cache`: Used to cache files to/from S3/GCS that help to expedite builds
+* `plugins/kaniko`: Used to build Docker images with the kaniko framework and push images to Docker registry out of the box
+* `plugins/kaniko-ecr`: Used to build Docker images with the kaniko framework and push images to AWS ECR registry out of the box
+* `plugins/kaniko-gcr`: Used to build Docker images with the kaniko framework and push images to GCP GCR registry out of the box
+* `plugins/s3`: Used to upload files to AWS S3 buckets out of the box
+* `plugins/gcs`: Used to upload files to GCP GCS service out of the box
```
Comment (aimurphy) [inline docs/continuous-integration/ci-technical-reference/harness-ci.md:37]: @ubekishov This is the content I added per DOC-18447. The images were already listed, and I added the descriptions.

File: docs/continuous-integration/ci-technical-reference/harness-ci.md [hunk 2]
```diff
@@ -10,35 +10,30 @@ helpdocs_is_private: false
 helpdocs_is_published: true
 ---
 
-The public Harness CI images on DockerHub needed for CI Pipelines are pulled automatically when you run your Harness Pipeline. You can find them at `https://hub.docker.com/u/harness`.
+The public Harness CI images on DockerHub that needed for CI Pipelines are pulled automatically when you run a Harness CI pipeline. You can find them a: (https://hub.docker.com/u/harness).
```
Comment (ubekishov) [inline docs/continuous-integration/ci-technical-reference/harness-ci.md:13]: at:

File: docs/continuous-integration/ci-technical-reference/harness-ci.md (hunk 1 above)
Comment (ubekishov) [reply docs/continuous-integration/ci-technical-reference/harness-ci.md:37]: Thank you. Descriptions are great

**Review by ubekishov — COMMENTED** [review]
Not sure if this is the right page, but by default we use Anonymous access to Dockerhub to pull these. You might run into rate limiting issues with this setup. To prevent that you just have to provide login information in the connector

**Comment by ubekishov** [comment]
Comments are added

File: docs/continuous-integration/ci-technical-reference/harness-ci.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Harness CI Images List
-description: The public Harness CI images on DockerHub needed for CI Pipelines are pulled automatically when you run your Harness Pipeline. You can find them at https &#8212; //hub.docker.com/u/harness. If you do not wan…
-tags: 
+title: Harness CI images list
+description: Public Harness CI images on DockerHub are pulled automatically when you run Harness CI pipelines.
+tags:
    - helpDocs
 # sidebar_position: 2
```
Comment (douglas-j-bothwell) [inline docs/continuous-integration/ci-technical-reference/harness-ci.md:6]: I recommend always setting the sidebar_position. You can also delete the `tags: `key and value.

File: docs/continuous-integration/ci-technical-reference/harness-ci.md (hunk 2 above)
Comment (aimurphy) [inline docs/continuous-integration/ci-technical-reference/harness-ci.md:13]:
```suggestion
The public Harness CI images on DockerHub that needed for CI Pipelines are pulled automatically when you run a Harness CI pipeline. You can find them at: (https://hub.docker.com/u/harness)
```

**Comment by ubekishov** [comment]
Maybe a comment on this page with the link to Docker registry. And then mention anonymous access on the Connector page 
(https://developer.harness.io/docs/platform/Connectors/connect-to-harness-container-image-registry-using-docker-connector).

**Review by douglas-j-bothwell — DISMISSED** [review]

File: docs/continuous-integration/ci-technical-reference/harness-ci.md [hunk 4]
```diff
@@ -1,44 +1,39 @@
...
+If you don't want the Harness Delegate to pull images from a public repo for security reasons, you can add a special **Harness Container Image Registry** connector to your Harness account. With this connector, the Delegate pulls these images from the **Harness Container Image Registry** only. For instructions on configuring this connector, go to [Connect to Harness Container Image Registry using Docker connector](../../platform/7_Connectors/connect-to-harness-container-image-registry-using-docker-connector.md).
 
-For steps on setting this up, see [Connect to Harness Container Image Registry Using Docker Connector](../../platform/7_Connectors/connect-to-harness-container-image-registry-using-docker-connector.md). 
+By default, Harness uses anonymous access to [Harness DockerHub](https://hub.docker.com/u/harness) to pull the images. If you experience rate limiting issues when pulling images, provide login information in the [Harness Container Image Registry Docker connector's authentication settings](https://developer.harness.io/docs/platform/Connectors/connect-to-harness-container-image-registry-using-docker-connector#step-2-enter-credentials).
```
Comment (aimurphy) [inline docs/continuous-integration/ci-technical-reference/harness-ci.md:15]: @ubekishov Is this ok?

Comment (ubekishov) [reply docs/continuous-integration/ci-technical-reference/harness-ci.md:15]: Looks great!

**Pull request merged**